react-frontend/src/listings/ListingView.js

```diff
@@ -280,6 +280,22 @@ function ListingView() {
     );
   }
 
+  const getBookmarkCount = () => {
+    let bookmarkCount;
```

Comment:
Consider initializing this value to zero, even if you are calling functions to get the numbers later.

Reply:
Good idea. Ill do that